content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -210,12 +210,17 @@ yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
 systemctl enable --now kubelet
 ```
 
-  **Note:**
+  **Notes:**
 
   - Setting SELinux in permissive mode by running `setenforce 0` and `sed ...` effectively disables it.
     This is required to allow containers to access the host filesystem, which is needed by pod networks for example.
+<<<<<<< HEAD
```

Comment:
@jfcgaspar ,
Can you clean up the extra text from the _rebase_? Thanks!

Reply:
@kbhawkey ,

Thanks for pointing it out, I think I removed it all.